client/src/components/layout/DestinationsIndex.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import DestinationTile from "./DestinationTile";
+
+const DestinationsIndex = (props) => {
+
+    const [destinations, setDestinations] = useState([])
+
+    const getDestinations = async () => {
+        try {
+            const response = await fetch("/api/v1/destinations")
+            if (!response.ok) {
+                const errorMessage = (`${response.status} ${response.statusText}`)
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const body = await response.json()
+            const newDestinations = body.destinations
+            setDestinations(
+                ...destinations,
```

Comment:
spread operator is not necessary as we are replacing an empty array with an array of objects. We would use the spread operator upon a POST request when we want to add one new element to an existing array of elements